web/controllers/v1/chat_handle_controller.ex

```diff
@@ -91,7 +91,7 @@ defmodule Cog.V1.ChatHandleController do
       chat_provider ->
         case Cog.chat_adapter_module(chat_provider.name) do
           {:ok, adapter} ->
-            case adapter.lookup_user(handle: handle) do
+            case adapter.lookup_user(handle) do
```

Comment:
Does this work with the slack adapter? I think we only match against `id: id` and `handle: handle`.